frontend/packages/ceph-storage-plugin/src/components/block-pool/body.tsx

```diff
@@ -0,0 +1,265 @@
+    storageClusterLoaded,
+    storageClusterLoadError,
+    state.isArbiterCluster,
+    dispatch,
+  ]);
+
+  const replicaList: string[] = _.keys(OCS_DEVICE_REPLICA).filter(
+    (replica: string) =>
+      (state.isArbiterCluster && replica === '4') || (!state.isArbiterCluster && replica !== '4'),
+  );
+
+  const replicaDropdownItems = replicaList.map((replica) => (
+    <DropdownItem
+      key={`replica-${OCS_DEVICE_REPLICA[replica]}`}
+      component="button"
+      id={replica}
+      data-test-id={replica}
+      onClick={(e) =>
+        dispatch({ type: BlockPoolActionType.SET_POOL_REPLICA_SIZE, payload: e.currentTarget.id })
+      }
+    >
+      {t('ceph-storage-plugin~{{replica}} Replication', { replica: OCS_DEVICE_REPLICA[replica] })}
+    </DropdownItem>
+  ));
+
+  return (
+    <>
+      {isClusterReady || !showPoolStatus ? (
+        <>
+          <div className="form-group ceph-block-pool-body__input">
+            <label className="control-label co-required" htmlFor="pool-name">
+              {t('ceph-storage-plugin~Pool Name')}
+            </label>
+            <input
+              className="pf-c-form-control"
+              type="text"
+              onChange={(e) =>
+                dispatch({
+                  type: BlockPoolActionType.SET_POOL_NAME,
+                  payload: e.currentTarget.value,
+                })
+              }
+              value={state.poolName}
+              placeholder={t('ceph-storage-plugin~my-storage-pool')}
+              aria-describedby={t('ceph-storage-plugin~pool-name-help')}
+              id="pool-name"
+              name="newPoolName"
+              data-test="new-pool-name-textbox"
+              required
+            />
+          </div>
+          <div className="form-group ceph-block-pool-body__input">
+            <label className="control-label co-required" htmlFor="pool-replica-size">
+              {t('ceph-storage-plugin~Data Protection Policy')}
+            </label>
+            <Dropdown
+              className="dropdown dropdown--full-width"
+              toggle={
+                <DropdownToggle
+                  id="replica-dropdown"
+                  data-test="replica-dropdown"
+                  onToggle={() => setReplicaOpen(!isReplicaOpen)}
+                  toggleIndicator={CaretDownIcon}
+                  isDisabled={state.isArbiterCluster}
+                >
+                  {state.replicaSize
+                    ? t('ceph-storage-plugin~{{replica}} Replication', {
+                        replica: OCS_DEVICE_REPLICA[state.replicaSize],
+                      })
+                    : t('ceph-storage-plugin~Select Replication')}
+                </DropdownToggle>
+              }
+              isOpen={isReplicaOpen}
+              dropdownItems={replicaDropdownItems}
+              onSelect={() => setReplicaOpen(false)}
+              id="pool-replica-size"
+            />
+          </div>
+          {isPoolManagementSupported && (
```

Comment:
If there are no items in dropdown, you must want to omit showing it.
Otherwise "create" would remain disabled.

Reply:
I am disabling create if volume type is empty, since it is a mandatory field  i can't hide this.

Reply:
I am disabling create if volume type is empty, since it is a mandatory field  i can't hide this.

Reply:
But in case volume type has no  items - we dont want to keep create disabled.
Or is it that it can never happen, I doubt :thinking:

Reply:
As I discussed with Sebastien, it never happens. Also, Eran wants this field as mandatory for pool creation.

Reply:
gr8 :+1: